ExploreASL_Import.m

```diff
@@ -556,7 +556,7 @@ function ExploreASL_Import(imPar, bCopySingleDicoms, bUseDCMTK, bCheckPermission
                                 [nii_files, scan_name, first_match, MsgDcm2nii] = xASL_io_dcm2nii(scanpath, destdir, scan_name, 'DicomFilter', imPar.dcmExtFilter, 'Verbose', imPar.bVerbose, 'Overwrite', imPar.bOverwrite, 'Version', imPar.dcm2nii_version, 'x', x);
 
                                 % If dcm2nii produced a warning or error, catch this & store it
-                                if ~isempty(MsgDcm2nii) && ~isempty(regexp(lower(MsgDcm2nii),'.*(warning|error).*')) % if it contains a warning/error
+                                if ~isempty(MsgDcm2nii) && ~isempty(regexp(lower(MsgDcm2nii),'.*(error).*')) % if it contains a warning/error
```

Comment:
better to use `regexpi()` instead of `regexp(lower())`

Reply:
Good catch. I have replaced that in the BIDS version only.

Reply:
Why not in both (if you want to sync them later anyway)?